### ISSUE TITLE
Accessibility: Pass checks and contrast for Message component.

### ISSUE THREAD

**Issue by EdwardCoyle** [infor-design/enterprise#964]
We need to check a handful of items related to accessibility on the Message Component.  Use the [HTML Code Sniffer](https://squizlabs.github.io/HTML_CodeSniffer/) tool or a similar one for detecting WCAG 2.0AA violations.  Make sure you have access to Windows environment with JAWS to test the announcer.

**Text color contrast issues**
NOTE: Available after #703 is merged.  Many of the examples on http://localhost:4000/components/message/example-index have contrast issues due to the lighter text color.  We should probably use a darker matching color, or black on the titles.  We may need to use a darker shade of blue on the primary buttons:

![screen shot 2018-10-09 at 5 13 26 pm](https://user-images.githubusercontent.com/3249601/46699049-aec71780-cbe6-11e8-9cdd-29907887c064.png)
![screen shot 2018-10-09 at 5 13 30 pm](https://user-images.githubusercontent.com/3249601/46699050-aec71780-cbe6-11e8-80e7-ef2883235186.png)

**Announcer Issues**
Double check how the announcer issues error messages.  It should properly announce that it's an error, as opposed to a regular message (@tmcconechy feel free to add notes about this one here).

To fix this look at the attributes... 
aria-labelledby
role

I think this should correctly announce that its an "alert", all of these are alerts of sorts. But it is only going to read whats in aria-labelled by. So "Application Error". So we might just want to check if the message type is in the string already, if its not add it in an audible label. For example if the message is "This record already exists", then add an audible label so it reads the type "Info: This record already exists" or "Error: This record already exists". Other than that i think its not too bad at the moment.

**Keyboard Control**
Ensure that while the message window is open, it's not possible to use the tab key to navigate to any items outside the scope of the Modal area (items beneath the overlay shouldn't be focusable without closing the message window).  Also check this in IE.  This isn't necessarily an issue now, but it's happened in the past and is worth re-checking / writing a e2e test for.

**Platform**
all

**Comment by EdwardCoyle** [comment]
Marking as [5] because installing the screen reader on a Windows environment will take a little time, if you haven't already got it installed.

**Comment by davidcarlsonberg** [comment]
@EdwardCoyle Do you happen to have a way to ensure JAWS reads `aria-labelledby`? Right now, the screen reader I have working on Windows8.1 IE11 is only reading the `message-title` and `message-text`.

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
